cardinal/server/utils/utils.go

```diff
@@ -1,7 +1,32 @@
 package utils
 
+import (
+	"reflect"
+)
+
 func GetQueryURL(group string, name string) string {
 	return "/query/" + group + "/" + name
 }
 
 func GetTxURL(group string, name string) string { return "/tx/" + group + "/" + name }
+
+// GetFieldInformation returns a map of the fields of a struct and their types.
+func GetFieldInformation(t reflect.Type) map[string]any {
```

Comment:
i think we should move this to a different package. this function isn't used anywhere in the server package

Reply:
moved to types